chat/basics/ios.md

```diff
@@ -5,21 +5,14 @@ title: Skygear Chat
 [[toc]]
 Skygear Chat is a collection of APIs to help you build Chat apps much easier. 
 
-## Enabling on Chat Plugin on Skygear Portal 
```

Comment:
Why do we remove this paragraph? I think our portal still have Chat disabled by default?

Reply:
Quick start also tells how to enable chat plugin , why duplicating?

Reply:
People might missed that and it is too easy to forget? Or at least make it looks like the Javascript counter part, that we reminds people to switch on the chat plugins in developer portal at Introduction again.

Reply:
I would include the link instead of pasting whole stuff again. By the way, why other platforms guides do not have such duplication?

Reply:
Javascript have it. Please make this consistent amongst all three platform in this pull request?